### ISSUE TITLE
Event Binding Issue

### ISSUE THREAD

**Issue by nspatilk** [thomasjbradley/signature-pad#170]
This is regarding below mentioned pull request.

https://github.com/nspatilk/signature-pad/pull/1

This pull request was raised because of below mentioned statement in jquery.signaturepad.js/ jquery.signaturepad-min.js.

$( “input” ).blur(); 

Essentially the above line means that it will look for all input element in the DOM and fire a blur event on to it. Even though signature element is not in focus blur event is getting triggered which is not correct.

Below is the proposed solution which will be specific binding. Blur event will be fired only when signature element is in focus.

$('input:focus').blur();

**Comment by thomasjbradley** [comment]
Yep, it's a good idea, unfortunately I won't be able to get the change in any time soon.
